.github/workflows/build.yml

```diff
@@ -100,10 +99,8 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Setup Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ env.NODEJS_VERSION }}
+      - name: Setup Node/NPM
```

Comment:
Should we add this to the other jobs? Feels like we should probably use the same version of node/npm for everything in a given build if possible

Reply:
By default the runner images are stable and come with stable node versions, for the purpose of the "Prepare" and "Publish" jobs I think that's pretty good. 

For E2E we intentionally pin it to different versions, no need to use volta for that since we aren't using the workspace version anyways.

Reply:
Got it. I was just trying to think of ways to reduce environmental differences between different CI jobs, just like we try to reduce the number of differences between CI environments and dev environments. Maybe it would make more sense if at some point we combined all of the work into a single job or something, anyway future optimizations I guess 😅